Azure/azure-cli
# [Storage] `az storage file/directory`: Add `--auth-mode login` and `--backup-intent` to support OAuth

**Related command**

`az storage directory` command group
`az storage file` command group
`az storage share list-handle/close-handle` commands

**Description**

File share OAuth is supported! Customers can use AAD role to call file share data plane commands with `--auth-mode login --backup-intent`.

PS. `--backup-intent`/`--enable-file-backup-request-intent` is required for using OAuth

**Testing Guide**

See scenario tests

---

This checklist is used to make sure that common guidelines for a pull request are followed.
